v3/src/components/map/hooks/use-map-model.ts

```diff
@@ -68,4 +66,13 @@ export function useMapModel(props: IProps) {
     mapModel.setHasBeenInitialized()
   }, [leafletMap, mapModel, mapModel.layers])
 
+  // Respond to content width and height changes
+  useEffect(function updateMapSize() {
+    const disposer = reaction(
+      () => [layout.mapWidth, layout.mapHeight],
+      () => leafletMap.invalidateSize(),
+      {name: "updateMapSize"}
```

Comment:
```suggestion
      {name: "MapContentModel.updateMapSize"}
```
so it's easier to know where to find it.